src/RCDEF/rcdef.cpp

```diff
@@ -3240,7 +3239,10 @@ LOCAL int gtoks(                 // Retrieve tokens from input stream "Fpm" acco
 						{
 							while ((c=fgetc(Fpm)) != '*')     // ignore til *
 								if (c == EOF)
-									goto ueof;
+								{
+									rcderr("Unexpected EOF: missing '*END'?");
+									return (gtokRet = GTEOF);
+								}
```

Comment:
This is not very DRY.

Reply:
Maybe I can create an inline function that contains the above code. Another possibility is instead of jumping back, we can jump forward after the last current return. My preferred method will be to create an inline function.

Reply:
Give it a try!

Reply:
Is this just to avoid a goto?  There are lots of gotos in CSE.  I would leave them alone unless you want to take the time to set up test cases and verify the changes work.  That kind of testing is time consuming because you have to construct cases that cause all the variants of the error.  "If it ain't broke ..."

Reply:
Try which of those options? I tried both before proposing them.